docker/bootstrap.sh

```diff
@@ -51,6 +63,12 @@ elif [ -x "/sytest/docker/${SYTEST_TARGET}_sytest.sh" ]; then
     exec "/sytest/docker/${SYTEST_TARGET}_sytest.sh" "$@"
 
 else
-    echo "sytest runner script for ${SYTEST_TARGET} not found" >&2
-    exit 1
+    PLUGIN_RUNNER=$(find /sytest/plugins/ -type f -exec test -x {} \; -name "${SYTEST_TARGET}_sytest.sh" -print)
```

Comment:
no need to shell out to `test` here

```suggestion
    PLUGIN_RUNNER=$(find /sytest/plugins/ -type f -executable -name "${SYTEST_TARGET}_sytest.sh" -print)
```

Reply:
-executable is not very portable to my understanding. e.g. FreeBSD find does not support it.

Reply:
macOS doesn't have it either.

Reply:
fair enough. Maybe we can just omit it? if it exists but isn't executable, that suggests an error rather than a file we should ignore?